src/coreclr/jit/jitconfigvalues.h

```diff
@@ -653,6 +653,9 @@ CONFIG_INTEGER(JitEnableHeadTailMerge, W("JitEnableHeadTailMerge"), 1)
 // Enable physical promotion
 CONFIG_INTEGER(JitEnablePhysicalPromotion, W("JitEnablePhysicalPromotion"), 1)
 
+// Enable cross-block local assertion prop
+CONFIG_INTEGER(JitDoCrossBlockLocalAssertionProp, W("JitDoCrossBlockLocalAssertionProp"), 0)
```

Comment:
`Do` seems fine, but should we use `Enable` instead, to match other similar cases (like `JitEnablePhysicalPromotion`, above)?

Reply:
Sure... it was up with the other `Do` cases before, but now that I moved it to be available in release it no longer matches its neighbors.